apiserver/facades/client/client/status.go

```diff
@@ -56,7 +56,9 @@ func (c *Client) FullStatus(ctx context.Context, args params.StatusParams) (para
 	}
 
 	var noStatus params.FullStatus
-	var context statusContext
+	context := statusContext{
+		applicationService: c.applicationService,
+	}
```

Comment:
OT: I hate these context structs, but considering this is going the way of the Dodo, we can allow it.